index.js

```diff
@@ -1,5 +1,5 @@
-module.exports = { 
-    Test: {
-        VlMap: require('./test/e2e/components/vl-map')
-    }
+module.exports = {
+  Test: {
+    VlMap: require('./test/e2e/components/vl-map'),
+  },
```

Comment:
ik blijf die trailing comma's een slecht idee vinden, maar ik weet niet wat de rest denkt?
@coemans @wautergu @SanderKleykens @Pieter-Beckers

Reply:
Ik zou ze zelf manueel nooit zetten, maar als ze gegenereerd worden heb ik er geen probleem mee.

Reply:
ik kan ze nog ergens begrijpen in lijsten/arrays maar om ze nu overal bij elke object literal op het einde te zetten zoals hier ...

Reply:
hier  bedoel ik dan bvb
```
const feature2 = new ol.Feature({
   geometry: new ol.geom.Point([109101, 204176]),
});
```

Reply:
Ik vind het persoonlijk niet erg. Dat het afgedwongen wordt, hoeft nu ook niet van mij, maar ik vind het geen probleem (en ergens ook gewoon een voordeel).

Reply:
Ik zou ze er ook niet zetten, maar ze werden gegenereerd.

Reply:
dan leg ik me neer bij de meerderheid, zij het met tegenzin :-)